### PR TITLE
changed landing url

### PR DESCRIPTION
**Pull request by meemansha07**
## Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Related Issue

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist:

- [ ] I have registered myself at [Contrihub website](https://sac.mnnit.ac.in/contrihub).
- [ ] My code follows the code style of this project.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

### CODE REVIEW

**Comment by eleensmathew** [comment]
@meemansha07 have you submitted the pr on the Contrihub website?

File: user/views.py [hunk 1]
```diff
@@ -13,11 +18,15 @@ def add_movie(request):
         title = request.POST.get('title')
         description = request.POST.get('description')
         poster = request.FILES.get('poster')
-        available = request.POST.get('available', False)
-        movie = Movies.objects.create(title=title, description=description, 
-poster=poster, available=available)
+        available = request.POST.get('available', False) == 'on'  # Correctly handle checkbox value
+        movie = Movies.objects.create(
```
Comment (eleensmathew) [inline user/views.py]: why is this code changed? and why have you added comments here?

File: user/views.py [hunk 2]
```diff
@@ -40,5 +48,5 @@ def post(self, request, *args, **kwargs):
         form = self.form_class(request.POST)
         if form.is_valid():
             form.save()
-            return redirect('user:shows')
+            return redirect(reverse('user:shows'))  # Use reverse to redirect to 'shows'
```
Comment (eleensmathew) [inline user/views.py]: why is this code changed?

**Comment by eleensmathew** [comment]
@meemansha07 an earlier pr was merged so make a git pull and commit again. Also pls fix the code based on comments.

**Comment by meemansha07** [comment]
> @meemansha07 an earlier pr was merged so make a git pull and commit again. Also pls fix the code based on comments.

Ok ma'am

**Pull request closed without merging**